packages/spacecat-shared-rum-api-client/src/index.js

```diff
@@ -88,9 +88,15 @@ async function generateDomainKey(domainkey, url, expiry) {
   return data[0].key;
 }
 
-async function createBacklink(dashboardUrl, domainKey, domainUrl, expiry) {
+async function createBacklink(dashboardUrl, domainKey, domainUrl, expiry, params) {
   const scopedDomainKey = await generateDomainKey(domainKey, domainUrl, expiry);
-  return `${dashboardUrl}?interval=${expiry}&offset=0&limit=100&url=${domainUrl}&domainkey=${scopedDomainKey}`;
+  return createUrl(dashboardUrl, {
+    offset: 0,
+    limit: 100,
```

Comment:
these values are hard-coded. should this be made configurable or selectable by the user of the client with sensible defaults?

Reply:
I think they can be made if they are passed in the params